src/routes/home/components/__tests__/DisciplineCard.spec.tsx

```diff
@@ -21,9 +21,7 @@ describe('DisciplineCard', () => {
 
   const renderDisciplineCard = (): RenderAPI =>
     render(
-      <NavigationContainer>
```

Comment:
@SteffiStuffel why did you have to add this container? The tests fail now :/
Instead i mocked the `react-navigation`. Is that fine? Seems like nothing breaks

Reply:
yes thats fine.